go-controller/pkg/util/net.go

```diff
@@ -97,7 +97,8 @@ func GetOVSPortMACAddress(portName string) (string, error) {
 
 // GetNodeWellKnownAddresses returns routerIP, Management Port IP and subnet mask
 // for a given subnet
-func GetNodeWellKnownAddresses(subnet *net.IPNet) (string, string, int, error) {
+
+func GetNodeWellKnownAddresses(subnet *net.IPNet) (string, string, int) {
```

Comment:
the added blank line breaks "make lint". but otherwise lgtm

Reply:
Doh. Thanks!